src/panels/profile/dialog-ha-mfa-module-setup-flow.ts

```diff
@@ -97,7 +97,7 @@ class HaMfaModuleSetupFlow extends LitElement {
                   ? html`<p>
                       ${this.hass.localize(
                         "ui.panel.profile.mfa_setup.step_done",
-                        { step: this._step.title }
+                        { step: this._step!.handler }
```

Comment:
handler is a technical key, why don't we get title anymore? I don't think this is the right fix.

Reply:
I agree with both your comments here. I spent entirely too long longer at this before eventually throwing in the towel and going with this approach, if not just to get a second opinion.

It looks like this was working (from looking at some YouTube videos) in 2018 and was then broken at some point in the years gone by - unnoticed - probably when there was an architecture change.

I thought too that we could probably, for now, safely just refer to a single translation as there is only one method.

Then, I found out that “Authenticator app” is hardcoded somewhere deep in the code (not translated)...

On your second point. This all started with me simply trying to take {qr_code} and wrap it with {url} to get a clickable method of enrolling MFA.

Don’t know if you’ve ever tried scanning a QR on mobile from your own screen (hint: it’s physically impossible, without another device).

With the current approach, I wasn’t able to get there :(

Reply:
> Don’t know if you’ve ever tried scanning a QR on mobile from your own screen (hint: it’s physically impossible, without another device).

I always make a screenshot and then run the image through a QR scanner 🙃

Reply:
I'll check what is available and what was changed soon

Reply:
Just checked and this is working.